Go/Go.sublime-syntax

```diff
@@ -118,25 +121,55 @@ contexts:
     #
     # //go:some_directive arg0 arg1 ...
     #
+    # These have been adopted outside the gc compiler for use in linting
+    # directives and pragmas for other compiler suites where they take the
+    # more generalised form of:
+    #
+    # //namespace:some_directive ...
+    #
     # They're not part of the language spec, may not be recognized by some Go
     # compilers, and may stop working in future releases. Therefore,
     # highlighting them as compiler pragmas could be misleading. We scope them
     # as plain comments by default, and add some detailed meta scopes for
     # enterprising users wishing to add more color.
-    - match: (//)(go)(:)({{ident}})?
+    - match: (//)([a-z]+[a-z-]*)(:)({{directive}})[ ]?
+      scope: meta.annotation.go
       captures:
         1: punctuation.definition.comment.go
         2: meta.keyword.annotation.go
-        3: meta.punctuation.accessor.colon.go
+        3: punctuation.accessor.colon.go
```

Comment:
I wonder why this `meta.` prefix is removed for the colon scope name here? This will result in only the colon to be highlighted and stand out by default. Shouldn't the `meta.` either be removed from the `meta.keyword`, `meta.variable.function` and `meta.variable.parameter` scopes as well then, or alternatively, if the directive comments shouldn't be highlighted, then keep it for the colon scope too?

Reply:
Absolutely correct! As stated in one of my comments above I was (am) tempted to remove those `meta.` scope prefix to enable highlighting such annotations because this is what we started by supporting comments with special meaning like `/// <directive>` in C#. If we don't want to highlight those, we don't need to add special patterns to scope them.

This change is a left over from tests related with those considerations.